### ISSUE TITLE
Add option for pulling in all test targets in a Podfile

### ISSUE THREAD

**Issue by jverkoey** [CocoaPods/CocoaPods#8273]
🌈

* [x] I've read and understood the [*CONTRIBUTING* guidelines and have done my best effort to follow](https://github.com/CocoaPods/CocoaPods/blob/master/CONTRIBUTING.md).

# Report

We're making use of the new test spec functionality, but finding it difficult to remember to maintain the testspecs list in our Podfile:

https://github.com/material-components/material-components-ios/blob/develop/catalog/Podfile#L8

Would it be possible to somehow provide an option for pulling in all tests for a given podspec?

**Comment by dnkoutso** [comment]
Yes! Look intohttps://github.com/CocoaPods/Core to do this. Would be a nice enhancement.

**Comment by amorde** [comment]
Sounds like a good idea to me 👍

**Comment by segiddins** [comment]
It'll require some plumbing to get to the resolver, most likely. That'll be the trickiest bit
